.github/workflows/golangci-lint.yml

```diff
@@ -13,9 +13,12 @@ jobs:
           - test
           - hack/tools
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
```

Comment:
This had to be added v2 was able to install its own go env, but v3 requires this explicitly.